lib/xverifier/verifier/applicator_with_options_builder.rb

```diff
@@ -0,0 +1,74 @@
+# frozen_string_literal: true
+
+module XVerifier
+  class Verifier
+    # Builds ApplicatorWithOptions from different invokation styles.
+    # @api private
+    # @attr base [Class]
+    #   class for which applicator_with_options should be built
+    # @attr args [Array]
+    #   array of arguments invoked with
```

Comment:
well, not in this place. I'll try to reword:
array of arguments `Verifier.verify` invoked with